### ISSUE TITLE
Web requests are routed using wrong method

### ISSUE THREAD

**Issue by ondrejchmelar** [faust-streaming/faust#188]
## Checklist

- [x] I have included information about relevant versions
- [ ] I have verified that the issue persists when using the `master` branch of Faust.

## Steps to reproduce

Create a class view with `@app.page` decorator and implement a `post` method as per https://faust.readthedocs.io/en/latest/userguide/tasks.html#http-verbs-get-post-put-delete

## Expected behavior

When running multiple workers, request is routed with POST method.

## Actual behavior

Requests to other workers are made using GET method.

Looks like GET is always used at https://github.com/faust-streaming/faust/blob/master/faust/app/router.py#L73

Could be as simple as calling `app.http_client.request(request.method, routed_url)`. 

# Versions

* Python version: CPython 3.9.5
* Faust version: v0.6.9
* Operating system: Linux x86_64
* Kafka version: kafka_2.13-2.7.0
* RocksDB version (if applicable): python-rocksdb==0.7.0

**Comment by patkivikram** [comment]
Yes the solution looks good! Can you submit a PR with your change?
